README.md

```diff
@@ -200,6 +201,58 @@ let numberOfWheels = 4
 let numberOfWheels = 4;
 ```
 
+### Imports
+
+Refer to [Import Declarations from Apple Doc].
+
+Imports should be declared at top of each Swift file and alphabetically (ignoraing case) sorted. There could be any number of comments before the first import sentence.
+
+Keep imports that declare a `kind` first ordered by their `kind` after imports that don't do it. Optionally group imports by kind declaration into logical blocks.
+
+Imports marked with `@testable` should be put at the end of the list of imports. Optionally you a trailing line may be put in order to group normal imports and `@testable` imports in two different logical blocks.
+
+Avoid duplicated imports.
+
+**prefered**
+```swift
+//
+//  AppDelegate.swift
+//  TestingHero
+//
+//  Created by ?? ??? on 1/1/17.
+//  Copyright © 2017 Xmartlabs. All rights reserved.
```

Comment:
Yup, 'm going to simplify it. I want to let clear that comments are allowed before imports